automata/symbol/graph/symbol_graph.py

```diff
@@ -40,6 +41,9 @@ def __init__(
         self._graph = builder.build_graph()
         self.navigator = SymbolGraphNavigator(self._graph)
 
+        with open("symbolgraph.pkl", "wb") as f:
```

Comment:
We should have the user set an optional path for where this will be written out to. For instance, we have `TASK_DB_PATH` in the .env, there could be a `SERIALIZED_GRAPH_PATH`.

Reply:
Further, there should be a flag that is passed to load the graph from this location if it exists.

Reply:
Next, we need to make this configurable all the way upstream to the CLI, so that a user can pass an argument like `--use-serialized-graph` if they do not want to reconstruct the graph

Reply:
Further, the graph data should live in the automata data repository (if anywhere), not in the main branch. That is why I proposed automata-embedding-data above.

Reply:
CLI integration is done—will be adding the reaming logic in #340